crawl-ref/source/dat/des/branches/shoals.des

```diff
@@ -2217,6 +2335,166 @@ swwwwwwwwwwwwwwww
 swwwwwwwwwwwwww
 ENDMAP
 
+NAME:  nicolae_shoals_bring_sunscreen
+TAGS:  transparent water_ok no_tide
+DEPTH: Shoals:2-
+KMONS: S = sun demon / sun moth
+SUBST: { = {[(, x = cxx..., S = S:7 .:6, _ = _.
+CLEAR: _
+MAP
+      _....._
+    _........._
+    ...S...S...
+   _..........._
+ _..S...xxx...S.._
+ ......xxcxx......
+_.......xxx......._
+..S..x.......x..S..
+....xxx.....xxx....
+....xcx..{..xcx....
+....xxx.....xxx....
+..S..x.......x..S..
+_.......xxx......._
+ ......xxcxx......
+ _..S...xxx...S.._
+   _..........._
+    ...S...S...
+    _........._
+      _....._
+ENDMAP
+
+NAME:   nicolae_shoals_offshore_bank_account
+TAGS:   water_ok no_pool_fixup transparent
+DEPTH:  Shoals
+ORIENT: west
+KITEM:  pr = $ w:100 / any jewellery
+KITEM:  q = $ w:50 / $ good_item w:30 / any jewellery good_item / \
+            any weapon good_item w:2 / any armour good_item w:2
+KMONS:  p = merfolk siren / nothing w:60
+KMONS:  q = merfolk impaler / merfolk javelineer / merfolk aquamancer / \
+            nothing w:30
+KMONS:  r = water nymph
+KFEAT:  ~ = open_sea
+KPROP:  =`pqr = no_tele_into
+KPROP:  =`pqr = no_tide
+KMASK:  =`pqr = opaque
+KMASK:  =`pqr = no_trap_gen
+KMASK:  =`pqr = no_monster_gen
+SUBST:  M = w_, N = wwW.
+CLEAR:  _
+MAP
+      MwwM
+    MwwwwwwM
+  MwwwwwwwwwwM
+~wwwwwwwwwwwwwwM
+~wwwwwwwwwwwwwwwwM
+~wwwwwcwwwcwwwwwwww
+~wwwwcccccccwwwwwNN
+~wwwcccqqqcccwwwN...
+~wwwwc`pppqcwwwN.....
+~wwww=`prpqcwwwN....@
+~wwwwc`pppqcwwwN.....
+~wwwcccqqqcccwwwN...
+~wwwwcccccccwwwwwNN
+~wwwwwcwwwcwwwwwwww
+~wwwwwwwwwwwwwwwwM
+~wwwwwwwwwwwwwwM
+  MwwwwwwwwwwM
+    MwwwwwwM
+      MwwM
+ENDMAP
+
+NAME:   nicolae_shoals_deep_elves_summer_vacation
+TAGS:   water_ok transparent patrolling no_pool_fixup
+DEPTH:  Shoals
+ORIENT: float
+# Someone left their gear in the changing room.
+KITEM:  P = leather armour / ring mail / scale mail / chain mail, \
+             long sword / short sword / rapier / dagger, \
+             buckler / nothing w:20
+# Some people like having a little something to read or drink on the beach.
+KITEM:  12 = nothing w:30 / any book / any scroll / any potion
+# Deep elves have to bring the high-strength sunscreen. At least one potion
+# of resistance is guaranteed in the changing room.
+KITEM:  s = potion of resistance ident:all
+# 1/3 chance of rudely walking in on a deep elf changing.
+KMONS:  p = deep elf knight ; nothing / nothing w:20
+KMONS:  1 = deep elf pyromancer w:20 / deep elf zephyrmancer w:20 / \
+            deep elf archer / deep elf knight / \
+            deep elf archer band w:5 / deep elf knight band w:5
+KMONS:  2 = deep elf death mage / deep elf sorcerer / \
```

Comment:
Vaguely wondering if this vault could have different themed sets for other branch vacationers. The elf set seems tough but manageable.

Reply:
> Vaguely wondering if this vault could have different themed sets for other branch vacationers.

It originally had nagas, although it was pointed out that that would somewhat dilute the distinctiveness of the s-branches, so I changed it to elves. Orcs would also work, but many players will have finished Orc before coming to the s-branches, so I'm not sure if that would be a sufficient challenge.